src/llmcompressor/transformers/finetune/runner.py

```diff
@@ -256,16 +261,13 @@ def run_sequential_stages(self, checkpoint: Optional[str] = None):
                 self.train(checkpoint=checkpoint, stage=stage_name)
             checkpoint = None
 
-            if (
-                self._training_args.output_dir
-                != TrainingArguments.__dataclass_fields__["output_dir"].default
-            ):
+            if self._training_args.output_dir:
```

Comment:
This check only checks if the user passes an empty string. If the user passes an empty string, the intended behavior should be that the user gets an error when they try to save (ideally earlier).

As coded now, we silently don't save the model. Please remove this line.

Reply:
self._training_args.output_dir always have a default value. it will save.